### DISCUSSION TITLE
Paula ‘Sampler Input’ possible?

### DISCUSSION THREAD

**Discussion by aZtOcKdOg** [vAmigaWeb/vAmigaWeb#180] [Ideas]
I wondered what code is in place in the base to utilise the audio device 'input / output' that is connected / in use by host device OS?

**Comment by mithrendal** [comment]
audio output is done via browsers AudioWorklet API 

for audio input I found this ...


https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Build_a_phone_with_peerjs/Connect_peers/Get_microphone_permission

so looks like it should be possible to connect Audio input stream to the Amiga VM ... when done all your other requested features ;-) then we can try this... I have been busy with other stuff and right now I am busy with first upgrading to latest vAmigaCore v2.4

**Reply by aZtOcKdOg** [reply]
Thanks for all ypur great work @mithrendal the setup has been extremely stable and useful for this Amiga game assets dev! Excellent! Much appreciated all your work thank you!
